stackslib/src/core/mempool.rs

```diff
@@ -803,6 +803,16 @@ const MEMPOOL_SCHEMA_6_NONCES: &'static [&'static str] = &[
     "#,
 ];
 
+const MEMPOOL_SCHEMA_7_TIME_ESTIMATES: &'static [&'static str] = &[
+    r#"
+    -- ALLOW NULL
+    ALTER TABLE mempool ADD COLUMN time_estimate_ms NUMBER;
+    "#,
+    r#"
+    INSERT INTO schema_version (version) VALUES (7)
```

Comment:
Should be `UPDATE schema_version SET version = 7` so we don't get a bunch of rows.

Reply:
Oh I see -- this is already a problem :/